### PR TITLE
Added a secton for invalid formatting examples

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@vikram-redhat PTAL

**Comment by vikram-redhat** [comment]
@mburke5678 does it render ok? If yes, then yeah looks good.

**Comment by mburke5678** [comment]
@vikram-redhat Not sure how to get `**`openshift-*`**` to render properly in the GitHub view (can't do asciibinder build). I tried a few dozen different placements of escape characters.

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -592,6 +592,17 @@ some code block
 
 . Item 3
 
+=== Invalid formatting
+
+The following are examples of invalid formatting and some resolutions.
+
+*Problem*
+\*\`openshift-*`* is invalid formatting. It is intended to mark up a literal value, `openshift-*`, in bold highlighting.
```
Comment (vikram-redhat) [inline contributing_to_docs/doc_guidelines.adoc]: escape all the special characters.

**Comment by mburke5678** [comment]
@vikram-redhat  Sorry. I wanted to put an example of double asterisks in the Solutions line. That one I cannot seem to get to work (can't get it to work here either!).

_or use double asterisks; `**`openshift-*`**`_ .

**Comment by vikram-redhat** [comment]
@mburke5678 - how we going with this?

**Comment by vikram-redhat** [comment]
Closing this in favor of: https://github.com/openshift/openshift-docs/pull/11268

**Pull request closed without merging**